parser/parser.go

```diff
@@ -51,6 +51,8 @@ func Parse(reader io.Reader) ([]Command, error) {
 
 			command.Name = string(fields[0])
 			command.Args = string(fields[1])
+		case "EMBED":
+			return nil, fmt.Errorf("deprecated command: %s", fields[0])
```

Comment:
Maybe a user friendly link that someone can follow to figure out how to do embeddings properly?